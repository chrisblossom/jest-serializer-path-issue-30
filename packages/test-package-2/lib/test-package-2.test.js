'use strict';

const testPackage2 = require('./test-package-2');

describe('testPackage2', () => {
    test('works', () => {
        const result = testPackage2();

        expect(result).toMatchSnapshot();
    });
});
