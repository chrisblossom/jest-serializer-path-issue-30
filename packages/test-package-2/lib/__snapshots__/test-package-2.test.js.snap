// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testPackage2 works 1`] = `"<PROJECT_ROOT>/packages/test-package-2/lib"`;
